libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -120,14 +120,15 @@ function(link_bc)
   endif()
 
   add_custom_command(
-    OUTPUT ${ARG_TARGET}.bc
-    COMMAND ${llvm-link_exe} ${link_flags} -o ${ARG_TARGET}.bc ${LINK_INPUT_ARG}
+    OUTPUT ${LIBCLC_ARCH_OBJFILE_DIR}/${ARG_TARGET}.bc
+    COMMAND ${llvm-link_exe} ${link_flags} -o ${LIBCLC_ARCH_OBJFILE_DIR}/${ARG_TARGET}.bc ${LINK_INPUT_ARG}
     DEPENDS ${llvm-link_target} ${ARG_DEPENDENCIES} ${ARG_INPUTS} ${RSP_FILE}
+    WORKING_DIRECTORY ${LIBCLC_ARCH_OBJFILE_DIR}
```

Comment:
do we need to set WORKING_DIRECTORY? IIUC all the command parameters and output is full path.

Reply:
Done, thanks